src/components/TaskBar/TaskBar.tsx

```diff
@@ -23,6 +23,11 @@ const ToolButton = styled.button`
   &:hover {
     color: #758bfd;
   }
+
+  &:disabled {
+    cursor: not-allowed;
```

Comment:
This is the style that was not working, right? In that case, you can remove this.

Reply:
I updated the code, this line needs to be in there for cursor to display the not-allowed style